docs/topics/proc_authenticating-the-oc-cli-client.adoc

```diff
@@ -10,7 +10,7 @@
 [id='authenticating-the-oc-cli-client_{context}']
 = Authenticating the `oc` CLI client
 
-To work with boosters on {parameter-deployment} using the `oc` command-line client, you need to authenticate the client using the token provided by the {parameter-deployment} web interface.
+To work with example applications on {parameter-deployment} using the `oc` command-line client, you need to authenticate the client using the token provided by the {parameter-deployment} web interface.
```

Comment:
you need to authenticate--> you must authenticate
The IBM Style Guide says: If an action must be performed, prefer “must,” not “need to.”

Reply:
Good catch. I also updated this in the modules for the other example applications.